docs/README.md

```diff
@@ -48,4 +48,4 @@ A quick reference to the commands that you will need when working locally.
 | `npm run test:client` | Run the client test suite. |
 | `npm run test:curriculum` | Run the curriculum test suite. |
 | `npm run test:server` | Run the server test suite. |
-| `npm run commit` | An interactive tool to help you build a good commit message. |
+| `npm run commit` | An interactive tool to help you build a good commit message. |
```

Comment:
Looks like nothing is being changed in this file. You should remove this from this commit.

```
git checkout upstream/master -- docs/README.md
```

Reply:
Done